lib/simp/cli/commands/bootstrap.rb

```diff
@@ -195,7 +195,7 @@ def self.configure_bootstrap_puppetserver
       # Run in a temporary cache space.
       server_conf_tmp = "#{::Utils.puppet_info[:config]['vardir']}/pserver_tmp"
       FileUtils.mkdir_p(server_conf_tmp)
-      FileUtils.chown('puppet','puppet',server_conf_tmp)
+      FileUtils.chown('pe-puppet','pe-puppet',server_conf_tmp)
```

Comment:
@bryanjbelanger Thanks for the patch, it identifies an issue that we need to discover between PE and FOSS versions of Puppet.

@simp/reviewers We need to make sure that we are discovering the user and group from the Utils.puppet_info[:config]` hash. Can we get a ticket created for this and hooked into a quick patch?